src/scss/objects/_form-controls.scss

```diff
@@ -0,0 +1,240 @@
+/**
+ * Custom Checkboxes and radios
+ *
+ * --------------------------------
+ *
+ * Checkbox button example HTML
+ *
+ *    <label class="c-form-label" for="checkbox1">
+ *        <input class="c-form-input" type="checkbox" id="checkbox1" name="myCheckbox" value="A value"/>
+ *        <span class="c-form-indicator c-form-indicator--checkbox"></span>
+ *        My Checkbox Label Text
+ *    </label>
+ *
+ * -------------------------------
+ *
+ * Radio button example HTML
+ *
+ *    <label class="c-form-label" for="radio1">
+ *        <input class="c-form-input" type="radio" id="radio1" name="myRadioBtn" value="A value"/>
+ *        <span class="c-form-indicator c-form-indicator--radio"></span>
+ *        My Radio Button Label Text
+ *     </label>
+ *
+ * ================================= */
+ $formControl-bgColor                  : $white !default;
+ $formControl-bgColor-disabled         : $color-disabled;
+ $formControl-borderColor              : $color-border;
+ $formControl-color             : $blue !default;
+ $formControl-background-color  : $color-bg !default;
+ $formControl-hover-color       : $blue--dark !default;
+ $formControl-borderColor-hover : $blue !default;
+ $formControl-margin-left       : 0 !default;
+ $formControl-padding-left      : spacing(x4) !default;
+ $formControl-width--wide       : 20px !default;
```

Comment:
I feel this should be 24px and then the `$formControl-width` should be 20px? Or am I misunderstanding the `--wide` bit?

Reply:
The wide modifier here is for a media query.  Might be worth adding a comment saying that though as agree could be confusing

Reply:
Ahh I see.